src/PKSim.Core/Model/RandomPopulationFactory.cs

```diff
@@ -165,17 +164,17 @@ private bool tryRandomize(Individual individual, RandomPopulationSettings popula
       }
 
       private void updateCurrentIndividualFromSettings(RandomPopulationSettings populationSettings, Individual currentIndividual,
-         PathCache<IDistributedParameter> allCurrentDistribuedParameters, PathCache<IDistributedParameter> allBaseDistributedParamters, Gender currentGender)
+         PathCache<IDistributedParameter> allCurrentDistribuedParameters, 
+         PathCache<IDistributedParameter> allBaseDistributedParamters, 
+         Gender currentGender,
+         RandomGenerator randomGenerator)
       {
          //clone of the original origin data
          currentIndividual.OriginData = populationSettings.BaseIndividual.OriginData.Clone();
          currentIndividual.OriginData.Gender = currentGender;
 
-         //create a new seed for the individual in order to randomize correctly our values
-         currentIndividual.GenerateSeed();
```

Comment:
@Yuri05 I have removed this call here. The RandomGenerator of the Population is now used using the seed